runtime/westend/src/lib.rs

```diff
@@ -1908,43 +1908,35 @@ mod remote_tests {
 	}
 }
 
-mod init_state_migration {
+mod clean_state_migration {
 	use super::Runtime;
 	use frame_support::traits::OnRuntimeUpgrade;
-	use pallet_state_trie_migration::{AutoLimits, MigrationLimits, MigrationProcess};
+	use pallet_state_trie_migration::{AutoLimits, MigrationProcess, SignedMigrationMaxLimits};
 	#[cfg(not(feature = "std"))]
 	use sp_std::prelude::*;
 
 	/// Initialize an automatic migration process.
-	pub struct InitMigrate;
-	impl OnRuntimeUpgrade for InitMigrate {
+	pub struct CleanMigrate;
+	impl OnRuntimeUpgrade for CleanMigrate {
 		#[cfg(feature = "try-runtime")]
 		fn pre_upgrade() -> Result<Vec<u8>, &'static str> {
-			frame_support::ensure!(
-				AutoLimits::<Runtime>::get().is_none(),
-				"Automigration already started."
-			);
 			Ok(Default::default())
 		}
 
 		fn on_runtime_upgrade() -> frame_support::weights::Weight {
-			if MigrationProcess::<Runtime>::get() == Default::default() &&
-				AutoLimits::<Runtime>::get().is_none()
-			{
-				AutoLimits::<Runtime>::put(Some(MigrationLimits { item: 160, size: 204800 }));
-				log::info!("Automatic trie migration started.");
-				<Runtime as frame_system::Config>::DbWeight::get().reads_writes(2, 1)
-			} else {
-				log::info!("Automatic trie migration not started.");
-				<Runtime as frame_system::Config>::DbWeight::get().reads(2)
-			}
+			MigrationProcess::<Runtime>::kill();
+			AutoLimits::<Runtime>::kill();
+			SignedMigrationMaxLimits::<Runtime>::kill();
```

Comment:
You can use `storage_alias` to have the definition of them without including the pallet.